### DISCUSSION TITLE
How to translate an LS-DYNA property subroutine to OpenRadioss

### DISCUSSION THREAD

**Discussion by Gadzbee** [OpenRadioss/OpenRadioss#2570] [Q&A] [answered]
Hello everyone,
I would like to implement a property subroutine to model muscle contraction behavior for muscle integration in a human model running with radioss. 

This subroutine already exists and has been developed by Kleinbach et al. in an open source way under LS-DYNA in the article : 

Kleinbach, Christian, Oleksandr Martynenko, Janik Promies, Daniel F. B. Haeufle, Jörg Fehr and Syn Schmitt. 2017. "Implementation and validation of the extended Hill-type muscle model with robust routing capabilities in LS-DYNA for active human body models". BioMedical Engineering OnLine, vol. 16, no. 1, p. 109. <https://doi.org/10.1186/s12938-017-0399-7>. 

I wanted to kno if there is a quick way to "translate" this code from LS-DYNA to OpenRadioss ?
If so, is it then possible to use this property in the HyperWorks environment ?

Thanks in advance.
Best regards,

Aurélien 

[Extended-Hill-type-Muscle-Model-for-LS-DYNA-v1.0.zip](https://github.com/OpenRadioss/OpenRadioss/files/15166010/Extended-Hill-type-Muscle-Model-for-LS-DYNA-v1.0.zip)

**Comment by MarianBulla** [comment, marked as answer]
Hello Aurelien,
it is quite easy to implement it into OpenRadioss.
Just follow the video on our OpenRadioss YouTube channel, how to create a user subroutine. 
For elements, it is the same process.

If you need a prototye, you can start with this implementation, please let me know.
I would love to see it implemented in OpenRadioss, by "the end of the day" :-)

Hope it helps you ?

Thanks and best regards,
Marian
